vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java

```diff
@@ -34,52 +32,47 @@ private LumoUtility() {
      */
     public static final class Accessibility {
 
-        private final LumoUtility dummyBackReference = null;
-
         /**
          * Hides content visually while keeping it available to screen readers.
          */
-        public static final String SCREEN_READER_ONLY = "sr-only";
+        public static final String SCREEN_READER_ONLY = notConstant("sr-only");
 
         private Accessibility() {
         }
 
     }
 
     /**
-     * Classes for distributing space around and between items along a flexbox’s
-     * cross axis or a grid’s block axis. Applies to flexbox and grid layouts.
+     * Classes for distributing space around and between items along a
+     * flexboxâ€™s cross axis or a gridâ€™s block axis. Applies to flexbox and
```

Comment:
Please revert this change, it contains some incorrect characters.

Reply:
Done and fixed